server/redisstate.go

```diff
@@ -194,8 +194,20 @@ func (s *RedisState) GetSenderOfTxHash(txHash string) (txSender string, found bo
 func (s *RedisState) AddTxToWhitehatBundle(bundleId string, signedTx string) error {
 	key := RedisKeyWhitehatBundleTransactions(bundleId)
 
+	// Check if item already exists
+	txs, err := s.GetWhitehatBundleTx(bundleId)
+	if err != nil {
+		return err
+	}
+
+	for _, tx := range txs {
+		if signedTx == tx {
+			return nil
```

Comment:
good to log the duplicate tx

Reply:
i think logging this will provide no benefit and just spam the logs.

Reply:
In particular because this is an expected case, that does not signal any kind of issue or problem.